scripts/kani-regression.sh

```diff
@@ -22,7 +22,7 @@ KANI_DIR=$SCRIPT_DIR/..
 export KANI_FAIL_ON_UNEXPECTED_DESCRIPTION="true"
 
 # Required dependencies
-check-cbmc-version.py --major 5 --minor 67
+check-cbmc-version.py --major 5 --minor 69
```

Comment:
I was just running into this (without updating cbmc you get a weird `goto-cc` failure in the test suite) so glad you snuck this in.